examples/chapter-list.lua

```diff
@@ -1,10 +1,19 @@
 --[[
     This script implements an interractive chapter list
-
```

Comment:
I don't want this line removed, the gap is part of my standard script header style.